Sources/App/Views/Sidebar/SidebarItem.swift

```diff
@@ -37,7 +47,7 @@ struct SidebarItem: View {
           Text(section.title)
```

Comment:
It's better to set the `foregroundColor` to all the `Text`s, `TextField`s, etc too 😄